### ISSUE TITLE
Form validation error

### ISSUE THREAD

**Issue by mayur20188** [seanCodes/bootstrap-less-port#20]
![bootstrap-less-form-error](https://user-images.githubusercontent.com/49226112/64060426-f7fecc00-cbe9-11e9-8494-32c2df099a12.png)

plz check img

**Comment by seanCodes** [comment]
Not sure what the issue is here... are you referring to the `#(` with the red squiggle underline? If so, that’s not an issue with this port but an issue with the VS Code Less syntax.

For reference, this is an [“anonymous mixin”](http://lesscss.org/functions/#list-functions-each):
> An anonymous mixin uses the form of `#()` or `.()` starting with `.` or `#` just like a regular mixin would.

Closing this but please let me know if I misunderstood or there’s another issue. Thanks!

**Comment by seanCodes** [comment]
@RossVertizan Have you customized your variables in any way or are you just using the default values? Could you try adding the flag `--math="parens-division"` when running on the command line and let me know if it helps?

**Comment by RossVertizan** [comment]
@seanCodes OK, I took a look at this again and in a clean environment so as not to confuse anything with my other settings.  here is what I did:
1. Cloned your GitHub repo to my local machine.

2. "yarn install".

3. ".\node_modules\.bin\lessc .\less\bootstrap.less bootstrap.css"
`RuntimeError: error evaluating function `each`: Operation on an invalid type in H:\git\bootstrap-less-port\less\_forms.less on line 247, column 1:                                                                                              246 //}                                                                                                                 247 each(@form-validation-states, #(@data, @state) {                                                                    248     @state-name: replace(@state, "@", "");    `

4. ".\node_modules\.bin\lessc --math="parens-division" .\less\bootstrap.less bootstrap.css"
... and .... (drum roll) this works.
